### DISCUSSION TITLE
How can I sync direcories in ~/.local/share/

### DISCUSSION THREAD

**Discussion by halloleo** [lra/mackup#1915] [General]
Some applications store their config in `~/.local/share/`. (Pandoc 3 for example - `~/.pandoc` is depreciated). Any way to sync these?

**Comment by joshmedeski** [comment]
Yep, you'll have to make a custom config file, here's an example of lollypop doing it https://github.com/lra/mackup/blob/3d2d41c3763820e3949c741536c1ced78fdcc214/mackup/applications/lollypop.cfg#L5

**Reply by halloleo** [reply]
Thank you. 🙏
